README.md

```diff
@@ -76,6 +76,12 @@ run `make install`.
 If you encounter compile errors, you could have out-dated versions of `grpcui`'s
 dependencies. You can update the dependencies by running `make updatedeps`.
 
+### Running without install 
+
+```
+go run ./cmd/grpcui/grpcui.go -plaintext localhost:9019
```

Comment:
It was not clear for me how to run the app locally, took me 10 minutes or so to figure out.